app/adapters/event.js

```diff
@@ -0,0 +1,19 @@
+import ApplicationAdapter from './application';
+import ENV from 'open-event-frontend/config/environment';
+import { computed } from '@ember/object';
+
+export default ApplicationAdapter.extend({
+
+  headers: computed('session.data.authenticated', 'session.currentRouteName', function() {
+    const headers = {
+      'Content-Type': 'application/vnd.api+json'
+    };
+    if (this.get('session.currentRouteName') !== 'public.index') {
```

Comment:
Not just index. Look for 

`this.get('router.currentRouteName').startsWith('public.')`